docs/de/index.rst

```diff
@@ -30,7 +30,8 @@ Die Dokumentation ist in anderen Sprachen verfügbar :
 
    user/faq
    user/installation
-   user/upgrade
+   user/upgrade-2.0.x-2.1.0
```

Comment:
Wow no.
I was talking about `user/upgrade-2.0.x-2.1.x` in case some user wants to upgrade from 2.0.6 to 2.1.3, you get it?